app/src/main/java/com/epmedu/animeal/tabs/TextInputField.kt

```diff
@@ -0,0 +1,313 @@
+package com.epmedu.animeal.tabs
+import android.content.res.Configuration.UI_MODE_NIGHT_NO
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement.Center
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.OffsetMapping
+import androidx.compose.ui.text.input.TransformedText
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.epmedu.animeal.base.theme.AnimealTheme
+import com.epmedu.animeal.base.theme.CursorColor
+import com.epmedu.animeal.base.theme.DarkerGrey
+import com.epmedu.animeal.base.theme.LynxWhite
+import com.epmedu.animeal.base.theme.NavigationItemColor
+
+@Composable
+fun TextInputField(
```

Comment:
Will you re-use `TextInputField` from previous merge request?

Reply:
No it is not reused, but I checked out new branch from 748 . 
because for example I added color in 748, I need that color in 749 so that I did like that .

Reply:
Fixed, I did seperately